### PR TITLE
Add QEMU

### PR DESCRIPTION
**Pull request by archeYR**
Add QEMU to rapps-db. Needs the -display sdl flag to display the screen correctly.
virtualbox_reactos_2_04_05_2018_19_58_55

### CODE REVIEW

File: qemu.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = QEMU
+Version = 2.7.0
+License = GPL v2
+Description = An open source and very fast machine emulator/virtualizer. It can emulate processors based on many various architectures (eg. x86, ARM, MIPS, PowerPC). Run with -display sdl flag to avoid the display glitches.
```
Comment (binarymaster) [inline qemu.txt]: Suggestion: You could use double quotes for `"-display sdl"`.

Comment (archeYR) [reply qemu.txt]: Done.

**Comment by learn-more** [comment]
What happens without the `-display sdl` flag?

**Comment by archeYR** [comment]
@learn-more Here is what happens without that flag.
virtualbox_reactos 2_12_06_2018_22_37_27
It is almost unusable then.

**Comment by learn-more** [comment]
I am not decided about this issue, due to it needing special commandline arguments for it to be usable.
Anyone care to weigh in?

**Comment by karlexceed** [comment]
I feel that though you can add a note about the command flag to the program description, it should be left out of RApps until this issue is resolved.

Unless:
- You could add something to the DB file so that RApps can set that flag or generate a .BAT launcher for it.
- RApps had the ability to set that flag / generate a BAT launch file.

**Comment by archeYR** [comment]
@karlexceed Including a batch file wouldn't really make a sense, as QEMU is meant to use with commandline. The notice about flag is already added to the description.

**Comment by archeYR** [comment]
Also, I want to point out that there is an older version of QEMU (from 2013) which doesn't have that problem, we can include the older version for now, and then upgrade when the issue with display is resolved.

**Comment by learn-more** [comment]
@archeYR That sounds like a winner.
Is there already an issue for this problem?

**Comment by archeYR** [comment]
@learn-more Alright then, I will change the QEMU version soon. There is no issue made specifically for QEMU, but it seems to be similar to [CORE-14589](https://jira.reactos.org/browse/CORE-14589)

**Comment by learn-more** [comment]
No worries, everyone does this in his spare time :)

**Pull request merged**